cmd/zfs_object_agent/server/src/main.rs

```diff
@@ -43,17 +43,27 @@ fn main() {
                 .help("Configuration file to set tunables (toml/json/yaml")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("log-config")
+                .short("l")
+                .long("log-config")
+                .value_name("FILE")
+                .help("Configuration yaml file logging")
```

Comment:
I think this is the right English word order?
```suggestion
                .help("Logging configuration yaml file")
```